filebeat/module/traefik/access/_meta/fields.yml

```diff
@@ -12,10 +12,12 @@
       description: >
         The number of requests
     - name: frontend_name
+      # Keyword?
       type: text
       description: >
         The name of the frontend used
     - name: backend_url
+      # Keyword? Or map to url.original
```

Comment:
Keyword, yes, absolutely.

But this doesn't map to url.original at all. This is the address used by Traefik to reach an upstream application server (e.g. `app1.myservice.local:3000`). Nothing to do with the actual web request URL :-)

Reply:
https://github.com/elastic/beats/pull/10401